### PR TITLE
Add tests for integer and optional primary keys for Swift

### PR DESCRIPTION
**Pull request by kishikawakatsumi**
For all integer types:
- Add tests for adding objects with primary key
- Add tests for adding objects with nil primary key
- Add tests for retrieve the object with primary key
- Add tests for retrieve the object with nil

Realm Objective-C already has integer primary key and null key.

CC @jpsim @mrackwitz

### CODE REVIEW

**Comment by austinzheng** [comment]
Looks good to me.

**Comment by jpsim** [comment]
👍

File: RealmSwift-swift2.0/Tests/RealmTests.swift [hunk 1]
```diff
@@ -496,17 +496,92 @@ class RealmTests: TestCase {
...
+            XCTAssertNil(missingObject)
+        }
+
+        func assertIntObject(objectType: Object.Type) {
+            let object = realm.objectForPrimaryKey(objectType, key: 1)
+            XCTAssertNotNil(object)
+
+            let missingObject = realm.objectForPrimaryKey(objectType, key: 0)
+            XCTAssertNil(missingObject)
+
```
Comment (mrackwitz) [inline RealmSwift-swift2.0/Tests/RealmTests.swift]: Superfluous empty line

Comment (kishikawakatsumi) [inline RealmSwift-swift2.0/Tests/RealmTests.swift]: Fixed, thanks.

**Comment by mrackwitz** [comment]
:+1:
